openpgp/write_test.go

```diff
@@ -498,6 +503,11 @@ func TestEncryption(t *testing.T) {
 		}
 
 		w, err := Encrypt(buf, kring[:1], signed, nil /* no hints */, config)
+		if err != nil && config.AEAD() != nil && !test.okV6 {
```

Comment:
I think here we should assert whether `(err != nil) == (test.okV6 && config.AEAD())`

Reply:
59054bcfe7b22dd6249705e188335ccfea5d6ac9